projects/azure/README.md

```diff
@@ -5,24 +5,184 @@ Main code repository is located in [Openshift Azure](https://github.com/openshif
 
 # Test CI-operator jobs 
 
-CI-Operator jobs are being triggered using [prow](https://github.com/kubernetes/test-infra/tree/master/prow)
-Prow configuration is located in this repository `ci-operator/jobs/openshift/openshift-azure/*.yaml`
+CI-Operator jobs are being triggered using [prow](https://github.com/kubernetes/test-infra/tree/master/prow).
+The Prow configuration is located in files within this repository at `ci-operator/jobs/openshift/openshift-azure/*.yaml`
 
-To run CI-Operator job manually you need to have to have [CI-Operator](https://github.com/openshift/ci-operator) installed in your path.
-Modify secret location in file `cluster-launch-e2e-azure.yaml` as below. This is because `ci-operator` set secret based on path where files are located and they are different in local development and CI server.
-``` - name: cluster-secrets-azure-file
-      secret:
-         secretName: azure
+To run a CI-Operator job manually it is recommended to use the [CI-Operator](https://github.com/openshift/ci-operator) container image
 ```
+docker pull registry.svc.ci.openshift.org/ci/ci-operator:latest
+```
+
+The templates used to run all of our ci-operator jobs (vm image builds and e2e tests) are located in 
+
+```
+ci-operator/templates/openshift/openshift-azure/
+```
+
+Obtain the `oc login` command from the OpenShift CI WebConsole and perform a login in your terminal
+
+Note: all further instructions assume you are currently in the root directory of this repository
 
-Populate secret file in `cluster/test-deploy/azure/secret` using `cluster/test-deploy/azure/secret_example` as template.
-If you are running this job not in CI cluster development namespace, but on other OpenShift cluster you will need to pre-populate base images in `azure` namespaces.
+Create a `secret` file for storing azure-related secrets using `cluster/test-deploy/azure/secret_example` as a template.
+
+```
+cp cluster/test-deploy/azure/secret_example cluster/test-deploy/azure/secret
+# insert/update credentials in cluster/test-deploy/azure/secret
+```
+
+Set the location of the namespace you wish to use for the ci-operator jobs
+```
+export CI_OPERATOR_NAMESPACE=your-chosen-namespace
+```
+
+#### Testing vm image builds
+
+Generate a new ssh keypair making sure to set the name of the keypair file as `ssh-privatekey`
+
+```
+ssh-keygen -t rsa -b 4096 -f cluster/test-deploy/azure/ssh-privatekey 
+```
+
+Checkout the repository containing the yum client certificates required by openshift-ansible to update the packages on the root VM.
+```
+git clone git@github.com:openshift/aos-ansible.git
```

Comment:
I'd suggest doing this cloning outside of the release repo.

Reply:
this was intended as a manual walk-through of the process of running a job manually and cleaning up afterwards. I think this, together with the cleanup section below is sufficient to show a developer everything needed to get this working. 

I also like that aos-ansible is inside the `release` root during the process so I can just go in and check stuff if i need to